src/SprykerSdk/Spryk/Console/SprykDumpConsole.php

```diff
@@ -30,24 +38,58 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output): void
     {
-        $sprykDefinitions = $this->getFacade()->getSprykDefinitions();
-        $sprykDefinitions = $this->formatForTable($sprykDefinitions);
+        $spryk = current((array)$input->getArgument(static::ARGUMENT_SPRYK));
```

Comment:
Can you extract methods here to make this method cleaner? I expect methods like `dumpOne()` and `dumpAll()`

Reply:
Done.